osidb/models/flaw/label.py

```diff
@@ -96,13 +96,17 @@ def create_from_affect(affect: Affect):
     def create_from_flaw(flaw: Flaw):
         """Add new labels to the flaw based on the flaw"""
 
-        [ps_modules, ps_components] = list(
+        flaw_affects = list(
             zip(
                 *Affect.objects.filter(flaw=flaw).values_list(
                     "ps_module", "ps_component"
                 )
             )
         )
+        if not flaw_affects:
+            return []
+
+        [ps_modules, ps_components] = flaw_affects
```

Comment:
I think the issue is in the unpacking. What about this change?

```
        ps_values = Affect.objects.filter(flaw=flaw).values_list("ps_module", "ps_component")

        if not ps_values:
            return []

        [ps_modules, ps_components] = list(zip(*ps_values))
```

Reply:
Great! I like your solution more, will change it, thanks!

Reply:
Glad to help! Could you please add a test once the PR is ready for review? Thanks :)